doc/source/tutorial/tutorial_transforming.ipyml

```diff
@@ -47,21 +47,15 @@ cells:
 
 
 - code: |
-    # replace all labels of the 'gender' axis by passing the list of all new labels
```

Comment:
I am unsure removing this entirely from the tutorial is a good thing because even if users should probably not do it, they will still want to do it and I fear (maybe this is unfounded) that if they don't find how to do it in the tutorial they will ask us. I guess we can try it this way and see what comes out of it.